demo/addons/fileimport/fileimport.js

```diff
@@ -309,6 +304,78 @@ const fileimport = (function () {
         $("#srs-select").append(srs.join(" "));
     }
 
+    var _unzip = function(file, oLayer) {
```

Comment:
Is it possible to separate the code in 3 files ?

main.js, csv.js, shp.js

Reply:
Yes good point, would be nice to have but I don't have the time today and will be less available the rest of the month due to holidays :-(

Reply:
Ok ;)